docs/develop/index.md

```
@@ -67,6 +79,32 @@ See [this file](https://github.com/Rindrics/mikazuki-munechika/blob/main/scripts
 
+### OpenTelemetry Setup (Optional)
+
+To enable local tracing with OpenTelemetry:
+
+1. Start the OTel infrastructure:
+
+```sh
+pnpm run otel:start
+```
+
+2. Add the following to your `.env.local`:
```

markdownlint-cli2 (0.18.1)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)